gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -476,6 +479,12 @@
   public static final String DEFAULT_JOB_HISTORY_STORE_PASSWORD = "gobblin";
 
   /**
+   * Password encryption and decryption properties.
+   */
+  public static final String ENCRYPT_KEY_FS_URI = "encrypt.key.fs.uri";
```

Comment:
I need to first get a `FileSystem` instance using fs uri, then call `FileSystem.exists()`, `FileSystem.open()`, etc. So I doubt I can combine them.